k-distribution/tests/regression-new/set-symbolic-tests/unsimplified/diff-12-spec.k

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2022 K Team. All Rights Reserved.
+// generated by gen-tests.sh
+
+requires "set-tests.k"
+
+module DIFF-12-SPEC
+    imports SET-TESTS
+
+    claim <k> diff ( SetItem(Y:MyId)      -Set (S SetItem(X:MyId)) )  => . </k> requires Y =/=K X
```

Comment:
The tests in `unsimplified` were kept  from the prior PR #2689 but are not expected to work because this PR does not add the relevant simplification rules (because of their performance/complexity impact).